### PR TITLE
add new blog entry for project portal

### PR DESCRIPTION
**Pull request by hollandjg**
new blog entry for project portal

Todo:
- [x] Make more positive – our challenges and learnings are pretty down on Gatsby. Let's include some positive stuff too.
- [x] fix styling of webpage images – two side-by-side on large screens, stacked on small screens.
- [x] content review from members of the team.
- [x] request content review by David Yokum
- [ ] content review by David Yokum
- [x] Date for release is past (wait for "go" from David)

### CODE REVIEW

File: blog/project-portal-completed.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+The benefits were compelling:
+* **All the code in one place**: A single repository makes it trivial to download and install the code for any and all of the sites. Deploying all the sites needed only a single interaction – triggering the GitHub Actions deployment script on the single repo.
+* **Low-effort changes**: Any changes made on one site would automatically be reflected on all the others.
+
+The drawbacks were major:
+* **Tight coupling**: By having all the code in a single repository and having all the sites deploy immediately whenever we merged code into the [main branch](https://www.atlassian.com/continuous-delivery/continuous-integration/trunk-based-development), we risked outages if, inevitably, a faulty bit of code made it through our pull request review process.
+* **Separation of concerns**: The single repository would get more and more unwieldy as new jurisdictions were added, and if one of them had totally different requirements (like having to be hosted on a different service) then we would have to fork the repository and maintain duplicate codebases.
```
Comment (jashlu) [inline blog/project-portal-completed.md:76]: Just a recommendation, would something like `Lack of modularization for scalability and customization` better capture the need to split the single repo into multiple repos?

**Review by jashlu — APPROVED** [review]

File: blog/project-portal-completed.md (hunk 1 above)
Comment (jashlu) [reply blog/project-portal-completed.md:76]: Addressing `Separation of concerns`, sorry it's not clearly referenced

File: blog/project-portal-completed.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+
+## CCV – from Prototype to Production
+
+CCV started work on the Project Portals in January 2022. [Mary McGrath](https://github.com/mcmcgrath13), [Joshua Lu](https://ccv.brown.edu/people/Joshua%20Lu), [John Holland](https://ccv.brown.edu/people/John%20Gerrard%20Holland), [Aisulu Omar](https://github.com/AisOmar), [Heather Yu](https://ccv.brown.edu/people/Heather%20Yu), [Jordan Lawson](https://ccv.brown.edu/people/Jordan%20Lawson) and [Joselynn Wallace](https://ccv.brown.edu/people/Joselynn%20Wallace) were involved over the course of our two-year engagement. 
+
+Our role was to help develop the website further:
+* Build a [Project Portal for the Government of North Carolina](https://projectportal.nc.gov),
```
Comment (jashlu) [inline blog/project-portal-completed.md:63]: I wonder if it makes sense to introduce San Antonio here as well, just cause Satx is referenced below in line 68

File: blog/project-portal-completed.md (hunk 1 above)
Comment (hollandjg) [reply blog/project-portal-completed.md:76]: That's a really good point. How about I summarize it as "limited scalability"? That feels to me like the fundamental thing here.

File: blog/project-portal-completed.md [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+
+### Gatsby is pretty hard to debug
+
+We found that Gatsby hides a lot of complexity from the designer, but that means also that a lot of things happen implicitly, or by convention, and often these things aren’t clearly documented. This makes debugging problems extremely difficult when they arise.
+
+## Ongoing Development and Support during 2023
+
+We met with the government users – all subject matter experts but not web developers – who would be updating projects on the sites to give them an introduction to the CMS. We gave each of them around an hour’s training, which turned out to be ample, and we only heard from them later if there were bugs in the CMS itself, or if they wanted changes to features. 
+
+Over the course of 2023 we continued to maintain and the portals – adding full text search, the ability to add a mailing-list signup, expanding the coverage of the CMS to manage every part of the site, and adding better image support.
```
Comment (jashlu) [inline blog/project-portal-completed.md:135]:
```suggestion
Over the course of 2023 we continued to maintain the portals – adding full text search, the ability to add a mailing-list signup, expanding the coverage of the CMS to manage every part of the site, and adding better image support.
```

Was this a typo?

File: blog/project-portal-completed.md [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+    width=1368 
+    height=488
+  />
+  <figcaption>
+    Data flow of the portal from storage on GitHub to the deployed site.
+  </figcaption>
+</figure>
+
+
+During the course of the development, we encountered some challenges, and learned some lessons.
```
Comment (jashlu) [inline blog/project-portal-completed.md:107]: Do you think this should be separated into a sub section or even its own section for "Our learnings" from the project or something like that. Just a thought

File: blog/project-portal-completed.md (hunk 1 above)
Comment (jashlu) [reply blog/project-portal-completed.md]: Love it!

**Comment by eldu** [comment]
@hollandjg did this ever get reviewed? Can we merge this in?

**Comment by eldu** [comment]
@hollandjg Pinging this PR again

**Comment by hollandjg** [comment]
> @hollandjg Pinging this PR again

Thanks for the ping. We're still waiting for David Yokum to issue a press release that the new lab has been founded at UNC, and he said we shouldn't publish anything until the new lab is announced. There doesn't seem to be anything on the UNC site yet.

**Pull request merged**